PyInstaller/utils/hooks/qt/__init__.py

```diff
@@ -809,6 +812,50 @@ def _collect_qtnetwork_openssl_macos(self, openssl_version):
 
         return binaries
 
+    def _collect_qtnetwork_openssl_linux(self, openssl_version):
+        """
+        Linux-specific collection of OpenSSL dylibs required by QtNetwork module.
+        """
+
+        # Out of the supported OSes, Linux is by most straight-forward, because OpenSSL shared libraries are expected
```

Comment:
I'm guessing there should be a `far the` after that `by`?

Reply:
Indeed.